atlasdb-config/src/test/java/com/palantir/atlasdb/config/AtlasDbConfigTest.java

```diff
@@ -20,16 +20,20 @@
 import static org.hamcrest.Matchers.nullValue;
 import static org.junit.Assert.assertThat;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 import java.util.Optional;
 
+import org.junit.BeforeClass;
 import org.junit.Test;
 
 import com.palantir.atlasdb.spi.KeyValueServiceConfig;
 import com.palantir.remoting2.config.ssl.SslConfiguration;
 
 public class AtlasDbConfigTest {
-    private static final KeyValueServiceConfig KVS_CONFIG = mock(KeyValueServiceConfig.class);
+    private static final KeyValueServiceConfig KVS_CONFIG_WITHOUT_NAMESPACE = mock(KeyValueServiceConfig.class);
+    private static final KeyValueServiceConfig KVS_CONFIG_WITH_RANDOM_NAMESPACE = mock(KeyValueServiceConfig.class);
```

Comment:
`RANDOM` here is a bit of a misnomer (the namespace name isn't random, despite literally being `random client`). How about `OTHER` (and `other client`)?